zerver/lib/message.py

```diff
@@ -1071,7 +1062,6 @@ def get_huddle_users(recipient_id: int) -> str:
             stream_dict[message_id] = dict(
                 stream_id=stream_id,
                 topic=topic,
-                sender_id=sender_id,
```

Comment:
I think we also need can change `get_raw_unread_data` to include `message__sender_id` in what is fetched, right?

Reply:
Oh, I guess not, since we need it to figure out what's going on with PMs.  OK.

Reply:
Yup, `message__sender_id` is used in `extract_unread_data_from_um_rows`.